test/Microsoft.AspNet.Mvc.FunctionalTests/TestHelper.cs

```diff
@@ -114,7 +117,10 @@ public static TestServer CreateServer(Action<IApplicationBuilder> builder, strin
                 applicationBasePath,
                 applicationWebSiteName);
             services.AddInstance<IApplicationEnvironment>(environment);
-            services.AddInstance<IHostingEnvironment>(new HostingEnvironment(environment));
+            var hostingEnvironment = new HostingEnvironment();
```

Comment:
probably a good idea.  that should avoid the ugly use here of `Microsoft.AspNet.Hosting.Internal` and `HostingUtilities`.